ui/main.qml

```diff
@@ -48,8 +48,6 @@ StatusWindow {
     property UtilsStore utilsStore: UtilsStore {}
 
     objectName: "mainWindow"
-    minimumWidth: 1200
-    minimumHeight: 680
```

Comment:
Just realized that removing the minimumWidth/Height will break the scaling on mobile, cause we use this function:

```js
    function updatePaddings() {
        if (applicationWindow.width < applicationWindow.minimumWidth) {
            const coefficient = applicationWindow.width / applicationWindow.minimumWidth;
            Theme.updatePaddings(Theme.defaultPadding * coefficient);
        } else {
            Theme.updatePaddings(Theme.defaultPadding);
        }
    }
```

Reply:
yeah, let's add the min width/height Theme. Something like portraitBreakpoint to replace all the usages of `minimumWidth` `minimumHeight`

Reply:
> yeah, let's add the min width/height to Theme. Something like portraitBreakpoint to replace all the usages of `minimumWidth` `minimumHeight`



> yeah, let's add the min width/height to Theme. Something like portraitBreakpoint to replace all the usages of `minimumWidth` `minimumHeight`

but isn't knowing the height and width enough to determine if its portrait mode or not? Maybe I need to check usages to understand better